### PR TITLE
Add all spoiled (official/unofficial) cards for SFK pack

### PR DESCRIPTION
**Pull request by zzorba**
Also sets release date to Friday

### CODE REVIEW

**Comment by zzorba** [comment]
Change takes all edits from https://github.com/Kamalisk/arkhamdb-json-data/pull/556 as well

File: pack/tde/sfk.json [hunk 1]
```diff
@@ -16,7 +16,86 @@
...
+		"traits": "Innate.",
+		"type_code": "skill",
+		"xp": 0
+	},
+	{
+		"code": "06112",
+		"cost": 2,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"illustrator": "German Nobile",
```
Comment (drawntotheflame) [inline pack/tde/sfk.json]: Switch to 'Germán Nóbile'

File: pack/tde/sfk.json [hunk 2]
```diff
@@ -16,7 +16,86 @@
...
+		"skill_willpower": 1,
+		"slot": "Hand",
+		"subname": "Untranslated",
+		"text": "[action]: Search your bonded cards for Essence of the Dream and add it to your hand.\n[reaction] After you succeed by 3 or more during a skill test to which Essence of the Dream is committed: Record in your Campaign Log that \"you have interpreted the dreams.\"",
+		"traits": "Item. Tome. Charm.",
+		"type_code": "asset",
+		"xp": 0
+	},
+	{
+		"code": "06113",
```
Comment (drawntotheflame) [inline pack/tde/sfk.json]: add: "bonded_to": 06112,

File: pack/tde/sfk.json [hunk 3]
```diff
@@ -34,13 +113,31 @@
...
+		"faction_code": "mystic",
+		"flavor": "A grisly future is laid before you.\nDo you turn away? Or do you embrace it?",
+		"illustrator": "Ilich Henriquez",
+		"name": "Read the Signs",
+		"pack_code": "sfk",
+		"position": 117,
+		"quantity": 2,
+		"skill_intellect": 1,
+		"skill_willpower": 1,
+		"text": "<b>Investigate.</b> Add yoru [willpower] value to your skill value for this investigation. You may ignore any effect or keyword on your location which would trigger during this investigation. If you succeed, discover 1 additional clue at this location.",
```
Comment (drawntotheflame) [inline pack/tde/sfk.json]: yoru -> your

**Comment by zzorba** [comment]
All fixed!  Ready to merge @Kamalisk

**Pull request merged**